src/omnibase_core/models/contracts/model_contract_effect.py

```
@@ -84,21 +84,77 @@ class ModelContractEffect(ModelContractBase):
+                    raise ModelOnexError(
+                        message=f"Invalid node_type: {v}",
+                        error_code=EnumCoreErrorCode.VALIDATION_ERROR,
+                        details=ModelErrorContext.with_context(
+                            {
+                                "error_type": ModelSchemaValue.from_value("valueerror"),
+                                "validation_context": ModelSchemaValue.from_value(
+                                    "model_validation",
+                                ),
+                            },
+                        ),
+                    )
```

Ruff (0.14.7)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)